specification.scs

```diff
@@ -0,0 +1,22 @@
+ims_kb_specification
+    <- concept_reusable_component_specification;;
+
+ims_kb_specification = [*
+concept_ims_ostis_kb
+    <- concept_reusable_component;
+    <- concept_nonatomic_reusable_component;
+    <- concept_independent_reusable_component;
+    <- concept_reusable_kb_component;
+    <- concept_reusable_dynamically_installed_component;
+    <- concept_reusable_source_code_component;
+
+    => nrel_sc_identifier: [IMS knowledge base specification](* <- lang_en;; *);
+    => nrel_key_sc_element: ... ;
+    => nrel_purpose: [IMS knowledge base is needed to ... ](* <- lang_en;; *);
+    => nrel_explanation: [Explanation] (*<- lang_en;; *);
+    => nrel_authors: ... ;
```

Comment:
It is **not** specification, it is _Knowledge base IMS_. Do not confuse a person with his passport. Identifier already exists in this kb. Purpose, explanation, authors, key element as well we do not need, because some kind of specification of _Knowledge base IMS_ already exists. Also it will be updated after OSTIS Standard translation. We just need to save part of specification, that is needed to install the component

Reply:
Then update your kb components, this was just copied by example

Reply:
Okay. 
In the ism-kb case I think we may specify identifiers, purpose, explanation, authors etc. (non-necessary specification part, but we need to do it correctly and gracefully, not just **...**,  **IMS knowledge base is needed to ...** or **Explanation**. Or specify gracefully or do not specify this part at all.

Reply:
That's why this pr was in draft, to discuss this part. I agree to not specify this right now cause we don't need this for sc-component-manger work.